src/engine/strat_engine/thinpool/mdv.rs

```diff
@@ -79,9 +81,15 @@ impl<'a> MountedMDV<'a> {
 
 impl<'a> Drop for MountedMDV<'a> {
     fn drop(&mut self) {
-        if let Err(err) = umount(&self.mdv.mount_pt) {
-            warn!("Could not unmount MDV: {}", err);
-        } else if let Err(err) = remove_dir(&self.mdv.mount_pt) {
+        for _ in 0..3 {
```

Comment:
Could you pull this one out and put it up for review in separate PR? I can approve it immediately.

Reply:
Resolved by #2942